kotlin/Deepgram-Topic-Detection/README.md

```diff
@@ -0,0 +1,63 @@
+# 🧹 Deepgram Transcribe Audio
+
+A Cloud Function for transcribing audio using [Deepgram](https://deepgram.com/)
```

Comment:
Would you please update this to be topic detection rather than transcribing audio?